Readme.md

```diff
@@ -0,0 +1,494 @@
+[1]: https://www.mongodb.com/atlas/database
+
+[2]: https://www.mongodb.com/docs/atlas/sample-data/
+
+[3]: https://www.mongodb.com/cloud/atlas/register
+
+[4]: https://www.mongodb.com/docs/drivers/kotlin/coroutine/current/
+
+[5]: https://github.com/mongodb-developer/kotlin-driver-quick-start
+
+# Getting started with MongoDB for building backend Applications using the official Kotlin driver
+
+> This is an introduction article on how to build backend application in Kotlin using official MongoDB Kotlin driver and [MongoDB Atlas][1].
+
+## Prerequisites
+
+This is a getting-started article, therefore nothing much is needed as a prerequisite, but familiarity with Kotlin as a programming language would be
+helpful.
+
+Also, we need an [Atlas account][3], which is free forever. Create an account if you haven't got one, that provides MongoDB as a cloud database and
+much more.
+
+In general, MongoDB is an open-source, cross-platform, and distributed document database that allows building apps with flexible schema. In
+case, you are not familiar with or would like a quick recap I would recommend exploring the MongoDB Jumpstart series to get familiar with MongoDB
+and its various services in under 10 minutes or if you prefer to read then you can follow
+this [guide](https://www.mongodb.com/docs/atlas/getting-started/).
+
+And last, to aid our development activities, we would be using [Jetbrains IntelliJ IDEA (Community Edition)](https://www.jetbrains.
+com/idea/download/).
+
+---------------------
+
+## Kotlin driver vs Realm Kotlin SDK ?
+
+Before we start I would like to touch base on [Realm Kotlin SDK](https://www.mongodb.com/docs/realm/sdk/kotlin/), which is another framework by
+MongoDB in Kotlin that is used to create client applications using the MongoDB ecosystem and shouldn't be confused with this. [MongoDB Kotlin driver]
+[4] is used to create backends applications like RESTful applications which can be consumed by client apps.
```

Comment:
I'd rephrase this a bit. The drivers are appropriate to create backends, scripts, etc. Restricting the Kotlin driver to just RESTful applications would be a disservice to the driver.